### PR TITLE
Added "the lounge" IRC client

### PR DESCRIPTION
**Pull request by passstab**
My first commit to the works list!

### CODE REVIEW

File: works.yaml [hunk 1]
```diff
@@ -1,4 +1,21 @@
 ---
+
+The Lounge:
+  uri:
+  - https://thelounge.github.io
+  description:
+  - The self-hosted web IRC client
```
Comment (apotheon) [inline works.yaml]: Please make this a complete sentence (preferably with the name of the work in it).

File: works.yaml [hunk 2]
```diff
@@ -1,4 +1,21 @@
...
+  license:
+  - MIT/X11 License
+  license_reference:
+  - https://github.com/thelounge/lounge
```
Comment (apotheon) [inline works.yaml]: Please link to the license text itself (e.g. a license file).

File: works.yaml [hunk 3]
```diff
@@ -1,4 +1,21 @@
 ---
+
```
Comment (apotheon) [inline works.yaml]: Please delete this blank line.

**Review by passstab — COMMENTED** [review]
is this better?

File: works.yaml [hunk 4]
```diff
@@ -5269,7 +5285,7 @@ aewm:
   license_reference:
   - http://freecode.com/projects/aewm
   description:
-  - "This is a modern, minimal window manager for X11. It is controlled entirely with the mouse, but contains no visible UI apart from window frames. The command set is sort of like vi: designed back in the dawn of time (1997), to squeeze speed out of low-memory machines, completely unintuitive and new-user-hostile, but quick and elegant in its own way."
+  - "aewm is a modern, minimal window manager for X11. It is controlled entirely with the mouse, but contains no visible UI apart from window frames. The command set is sort of like vi: designed back in the dawn of time (1997), to squeeze speed out of low-memory machines, completely unintuitive and new-user-hostile, but quick and elegant in its own way."
```
Comment (apotheon) [inline works.yaml]: This change is not necessary.  The first letter of the sentence should be capitalized, and aewm does not appear to be a capitalized term, so starting the sentence with "This" is preferable to "aewm" or "Aewm".

Comment (apotheon) [reply works.yaml]: If you want to rework the sentence so it starts with a capitalized term with "aewm" somewhere in the text, that's fine.  There are some examples of this approach, usually in the form "The $foo project provides . . ." or something along those lines.

**Comment by apotheon** [comment]
Excellent.  Thank you for the contribution.

**Pull request merged**